WORKSPACE

```diff
@@ -44,11 +44,10 @@ http_archive(
 
 http_archive(
     name = "io_bazel_rules_docker",
-    sha256 = "413bb1ec0895a8d3249a01edf24b82fd06af3c8633c9fb833a0cb1d4b234d46d",
-    strip_prefix = "rules_docker-0.12.0",
+    sha256 = "7d453450e1eb70e238eea6b31f4115607ec1200e91afea01c25f9804f37e39c8",
```

Comment:
We will need to find out what is causing the issue. That will for instance conflict with #2975 which brings in a rules_go fix.

Reply:
This should not be an issue anymore in the new patch, I was able to maintain the same rules_docker release.